src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -1078,11 +1079,61 @@ public void Select(int start, int length)
 
 		public void SelectAll() => SelectAllPartial();
 
-
 		partial void SelectPartial(int start, int length);
 
 		partial void SelectAllPartial();
 
+		/// <summary>
+		/// Copies content from the OS clipboard into the text control.
+		/// </summary>
+		public async void PasteFromClipboard()
```

Comment:
Being `async`, I think this doesn't match UWP/WinUI? Is that a problem?

Reply:
I need to document this, but it will need to stay async due to the fact that WASM uses async for clipboard methods. On the other hand it should not be a huge problem as long as developers are aware that it might take a few ticks before the value is actually pasted.

Reply:
`async void` should really be avoided because it cannot be awaited.

If it's possible to diverge from UWP/WinUI here, I'd make it `async Task` instead. Otherwise, try with approaches like `Task.Run(async () => await content.GetTextAsync()).Wait()`.

Reply:
We should rather dispatch it again on the dispatcher than use `async void`.